specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2018-07-10/service.json

```diff
@@ -8043,6 +8051,10 @@
         "recoveryAvailabilityZone": {
           "description": "The recovery availability zone.",
           "type": "string"
+        },
+        "vmEncryptionType": {
```

Comment:
does it have an allowed set of values? can we make this enum?

Reply:
Changed this and added a set of allowed values.